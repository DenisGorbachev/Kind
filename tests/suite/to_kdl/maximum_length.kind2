MyFn #WowThisNameIsSoLongCantFitOnKdl : U60 {
  10
}