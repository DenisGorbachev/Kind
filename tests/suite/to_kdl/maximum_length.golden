Kindelia name "WowThisNameIsSoLongCantFitOnKdl" for "MyFn" has more than 11 characters.
